web-registry/src/lib/ecocredit/constants.ts

```diff
@@ -138,6 +139,10 @@ export const ECOCREDIT_MESSAGE_TYPES = {
     message: `/${MsgBridge.$type}`,
     readable: 'bridge',
   },
+  BRIDGE_RECEIVE: {
```

Comment:
We won't be testing this as part of this PR, but I am adding `MsgBridgeReceive` to the activity table here before I forget.

Reply:
`MsgBridge` is already in here, so you will see that in /activity if a TX succeeds.